org-roam.el

```diff
@@ -1680,6 +1700,72 @@ command will offer you to create one."
       (when (y-or-n-p "Index file does not exist.  Would you like to create it? ")
         (org-roam-find-file "Index")))))
 
+;;;###autoload
+(defun org-roam-alias-add ()
```

Comment:
Not sure about naming of these functions, so if you have a better names in mind, please let me know 😸

Reply:
Names are fine, I think.